### PR TITLE
Add FR-NL-3.04, FR-NL-4.01 and FR-NL-5.01

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Reason for change
Added FR-NL-3.04 period dates must not include time
Added FR-NL-4.01 Forever period must not be used
Added FR-NL-5.01 Unused context not allowed

#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: tests/resources/conformance_suites/nl_nt16/fr_nl/3-04-duplicate-contexts-dimension.xbrl [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  xmlns:jenv-bw2-dim="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-axes"
+  xmlns:jenv-bw2-dm="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-domains"
+  xmlns:jenv-bw2-i="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-data"
+  xmlns:link="http://www.xbrl.org/2003/linkbase"
+  xmlns:xbrldi="http://xbrl.org/2006/xbrldi"
+  xmlns:xbrli="http://www.xbrl.org/2003/instance"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+    <link:schemaRef xlink:href="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-data.xsd" xlink:type="simple"/>
+
+	<context id="ctx-3A">
```
Comment (aaroncameron-wk) [inline tests/resources/conformance_suites/nl_nt16/fr_nl/3-04-duplicate-contexts-dimension.xbrl:15]: Nit: Mix of tabs and spaces in some of these test case files. Also, some missing new lines at EOF.

**Review by aaroncameron-wk — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Comment by allenbross-wf** [comment]
QA  +10 All tests passed.

File: tests/resources/conformance_suites/nl_nt16/fr_nl/3-04-duplicate-contexts-dimension.xbrl [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  xmlns:jenv-bw2-i="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-data"
+  xmlns:link="http://www.xbrl.org/2003/linkbase"
+  xmlns:xbrldi="http://xbrl.org/2006/xbrldi"
+  xmlns:xbrli="http://www.xbrl.org/2003/instance"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <link:schemaRef xlink:href="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-data.xsd" xlink:type="simple"/>
+
+	<context id="ctx-3A">
+	  <entity>
+		<identifier scheme="ttp://www.kvk.nl/kvk-id">0001445305</identifier>
```
[the remark this answers is not in the document]
Comment (derekgengenbacher-wf) [reply tests/resources/conformance_suites/nl_nt16/fr_nl/3-04-duplicate-contexts-dimension.xbrl:17]: Allen is going to fix in his last validation PR. Nice find!

**Pull request merged**